part-2-server/server.js

```diff
@@ -0,0 +1,35 @@
+var http = require('http');
+var fs = require('fs');
+
+var server = 
+	http.createServer(function(req, res) {
+		var url = req.url;
+		// ### Step 1:
+		if(url === '/') {
+			fs.readFile('./index.html', function(err, data) {
+				if(err) { throw err }
+				res.write(data.toString());
+				res.end();
+			})
+		}
+		// ### Step 2:
+		if (url === '/page-2' || url === '/page-3') {
+			fs.readFile('./page-2.html', function(err, data) {
+				if(err) { throw err }
+				res.write(data.toString());
+				res.end();
+			})
+		}
+		// ### Step 4:
```

Comment:
It'll be a little inefficient since you need to run this for loop for every request....is there a way you can do this more efficiently, without using a loop at all?